sbat.c

```diff
@@ -171,11 +172,14 @@ verify_sbat_helper(list_t *local_sbat_var, size_t n, struct sbat_section_entry *
 	}
 
 	for (i = 0; i < n; i++) {
+		bool found = false;
```

Comment:
Mmm this can go one block further inside?

Reply:
Sure.